slices/binutils-aarch64-linux-gnu.yaml

```
@@ -0,0 +1,87 @@
+package: binutils-aarch64-linux-gnu
+
+essential:
+  - binutils-aarch64-linux-gnu_copyright
+
+slices:
+  # NOTE: You might want to create a symlink from /usr/bin/as to
+  #       /usr/bin/aarch64-linux-gnu-as in your rootfs to use these slices.
+  #       ln -s "/usr/bin/${arch}-as" "${rootfs}/usr/bin/as"
+  assembler:
+    essential:
+      - libbinutils_libs
+      - libc6_libs
+      - libzstd1_libs
+      - zlib1g_libs
+    contents:
+      /usr/bin/aarch64-linux-gnu-as:
```

Relevant to the comment above, what about the symlinks for these binaries? It's very nice to be able to prepend `$PATH` for cases like this.

```suggestion
      /usr/bin/aarch64-linux-gnu-as:
      /usr/aarch64-linux-gnu/bin/as:
```